apps/frontend-manage/src/components/evaluation/TableChart.tsx

```diff
@@ -25,27 +25,23 @@ function TableChart({
 }: TableChartProps): React.ReactElement {
   const ref = useRef<{ reset: () => void }>(null)
 
+  const formatCorrect = (value: boolean | undefined | null): string => {
+    if (value === true) return 'T'
+    if (value === false) return 'F'
+    return '--'
+  }
+
   const tableData = useMemo(() => {
     if (QUESTION_GROUPS.CHOICES.includes(data.questionData.type)) {
       return (data.questionData as ChoicesQuestionData).options.choices.map(
         (choice: Choice, index: number) => {
           return {
             count: data.results[index].count,
-            value: (
-              <Markdown
-                content={choice.value}
-                className={{ img: 'max-h-32' }}
-              />
-            ),
-            correct: choice.correct ? 'T' : 'F',
+            value: choice.value,
+            correct: formatCorrect(choice.correct),
             percentage: data.participants
-              ? String(
-                  (
-                    (data.results[index].count / data.participants) *
-                    100
-                  ).toFixed()
-                ) + ' %'
-              : '0 %',
+              ? data.results[index].count / data.participants
```

Comment:
this would be a case for a transformer?

Reply:
transformers and formatters should get the value of the column first, but also access to the entire row, so that such calculations can be done within them and not on a higher-level.